src/update-path.js

```diff
@@ -15,12 +15,33 @@ const updateAbsolutePath = async (hostSite, filePath, outputPath) => {
       for (var i = 0; i < length; i++) {
         const { name, value } = node.attributes[i];
         // do stuff...
+        var urlPattern = /^(?:\/|[a-z]+:\/\/)/;
+        if(!urlPattern.test(node.attributes[i].value)){
+          node.attributes[i].value = absolute(hostSite,node.attributes[i].value);
+        }
       }
     }
   });
   // save file to outputPath
   // ...
+  jsdom.writeFile(outputPath,window.document.documentElement.outerHTML);
```

Comment:
Add trailing space after `,`
Install & config prettier plugin to enable auto format code.

Reply:
okie